MoreLinq.Test/FullGroupJoinTest.cs

```diff
@@ -125,6 +126,18 @@ public void FullGroupPreservesOrder()
                 res.Second.AssertSequenceEqual(listB.Where(t => t.Item1 == res.Key).ToArray());
             }
         }
+
+        private static IEnumerable<(int Key, IEnumerable<T> First, IEnumerable<T> Second)> DoJoin<T>(OverloadCase overloadCase, IEnumerable<T> listA, IEnumerable<T> listB, Func<T, int> getKey)
+        {
+            switch (overloadCase) {
+            case OverloadCase.ResultSelector:
+                return listA.FullGroupJoin(listB, getKey, getKey, (k, f, s) => (k, f, s));
```

Comment:
Do you prefer spelling it out as `(k, f, s) => (k, f, s)` instead of just using `TupleValue.Create` as a method group?